doc/manual.rst

```diff
@@ -1013,7 +1015,9 @@ intXX
   Literals of these types have the suffix 'iXX.
 
 `uint`
-  the generic `unsigned integer`:idx: type; its size is platform-dependent and has the same size as a pointer. An integer literal with the type suffix `'u` is of this type.
+  the generic `unsigned integer`:idx: type; its size is platform-dependent and
+  has the same size as a pointer. An integer literal with the type
```

Comment:
```suggestion
  the same as the size of a pointer. An integer literal with the type
```

Reply:
"its size is platform-dependent and has the same size as a pointer" means "its size has the same size as a pointer", which doesn't make sense. An alternate formulation would be
```suggestion
  it has the same size as a pointer. An integer literal with the type
```